apps/extract_step/lib/extract/message.ex

```diff
@@ -0,0 +1,29 @@
+defmodule Extract.Message do
```

Comment:
do these ever get persisted anywhere? like in brook state or are they _totally_ for in-transit messages? i ask b/c not using the `Definition` module to allow for all of the lifecycle management of the structs could blow up in our faces if they're persisted anywhere.

Reply:
totally in-transit